pkg/core/index_shards.go

```diff
@@ -105,14 +104,6 @@ func (s *IndexShard) GetIndexName() string {
 }
 
 func (s *IndexShard) GetShardName() string {
```

Comment:
What is the intention of the code?

Reply:
shard name used to register WAL consume list. and will used for distribution by doc id.